src/app/homepage/homepage.component.scss

```diff
@@ -0,0 +1,85 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@mixin theme($theme) {
+  .recent-container {
+    height: 40vh;
```

Comment:
This causes a lot of blank space between Recent and Your Sets on my screen 
<img width="1383" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/e37a2919-c981-4d36-a3da-c1547845de0d">

Try a different way of spacing/defining the container's height